### ISSUE TITLE
Fix conflict between internal dummy label `99` and user-supplied label `99`

### ISSUE THREAD

**Issue by jcohenadad** [spinalcordtoolbox/spinalcordtoolbox#4228]
context: https://forum.spinalcordmri.org/t/registration-of-lumbar-data-to-pam50/1109/5

suggestions:
- we check for input label 99 in custom label, we update the doc to make it clear
- we use an alternative approach for creating our dummy label internally (to make sure no conflict is possible).

**Comment by joshuacwnewton** [comment]
Which solution to apply depends on how the value `99` is used: 

- Is it hardcoded elsewhere? i.e. Do we have code that specifically looks for the value `99`? 
- Or does the specific number not matter at all? (i.e. `100`, `101`, `102` would all produce the same results)

This is easily answered by looking at the outer context:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/792a155fdb55aaae8096a2d6b32874d1a98fcc13/spinalcordtoolbox/scripts/sct_register_to_template.py#L780-L782

The same dummy label is added to both the subject label image and the template label image -- meaning that the value truly doesn't matter at all, as long it doesn't conflict with the existing labels. (To double-check, I did a search in our codebase, and I can't seem to find any other mention of `99` beyond the `add_orthogonal_label` function.)

I then tried testing `100`, and got the same "fixed rotation issues" results as @jcohenadad got by [changing the user's labels to 97](https://forum.spinalcordmri.org/t/registration-of-lumbar-data-to-pam50/1109/5#ref-subject-3).

So, I think it should be fine to just check for existence of `99` then using a different label in that case.
